Library/Formula/gearman.rb

```diff
@@ -107,7 +107,7 @@ def plist; <<-EOS.undent
 +++ b/libgearman-1.0/gearman.h
 @@ -50,7 +50,11 @@
  #endif
- 
+
```

Comment:
whitespace is significant in patches (the first column is always either a space, a `+`, or a `-`, and trailing whitespace is present in the source lines which provide context for the patch)